pkg/sql/parse.go

```diff
@@ -69,7 +69,7 @@ func parse(dbms, sqlProgram string) ([]statementParseResult, error) {
 		return sqls, nil
 	}
 
-	left := sqlProgram[:i]
+	left := sqls[len(sqls)-1].standard
```

Comment:
In the case of `SELECT ...; SELECT * FROM my_table TO TRAIN ...`, `sqlProgram[:i]` is `SELECT ...; SELECT * FROM my_table`, while it should be `SELECT * FROM my_table`.